assets/admin/students/student-bulk-action-button/index.js

```diff
@@ -38,7 +42,21 @@ export const StudentBulkActionButton = () => {
 			default:
 		}
 	};
-
+	const buttonEnableDisableEventHandler = ( args ) => {
+		setButtonDisabled( ! ( args.detail && args.detail.enable ) );
+	};
+	useEffect( () => {
+		document.addEventListener(
```

Comment:
There is a linter warning recommending us to use global, instead of the document.
<img width="799" alt="Screen Shot 2022-04-27 at 14 21 53" src="https://user-images.githubusercontent.com/38718/165583817-ef95c5a0-2c85-472e-bfd0-c432acee52cc.png">


```suggestion
		global.addEventListener(
```